### DISCUSSION TITLE
Is there some way to package nicegui's program and run it as exe instead of *.py ?

### DISCUSSION THREAD

**Discussion by bigantcgg** [zauberzeug/nicegui#724] [Q&A]
When I package nicegui into exe through pyinstaller, it cannot run successfully on win10.
Is there a special way to package it?
Or does nicegui allow packaging into exe to run?

**Comment by falkoschindler** [comment]
This should actually work. Did you follow the [documentation](https://nicegui.io/documentation#package_for_installation)?
Versions 1.2.4 and 1.2.5 had a bit of a hiccup on Windows, but 1.2.6 should work again.
If you keep having problems, please describe what is happening and what exactly is not working. Thanks!

**Reply by bigantcgg** [reply]
thank you for your reply.
The environment I use is win10 + anaconda[env: python3.10]
Using python main.py in env is no problem.

The exe generated by pyinstaller -F main.py cannot be run successfully.

The exe generated by python build.py also cannot run successfully. And its error is what is shown in this image below.
![Snipaste_2023-04-06_16-22-37](https://user-images.githubusercontent.com/129948909/230361209-e24f5165-f829-466c-ba86-8551fb139241.png)


Thanks again.

**Reply by falkoschindler** [reply]
It looks like a problem with allocating enough memory. Do you have enough free RAM available?
But it's strange, because the error happens when deserializing the page layout string "HHH LpR FFF". That doesn't look very memory hungry.

**Reply by bigantcgg** [reply]
When running the exe generated by build.py, a background process will be automatically generated
![Snipaste_2023-04-10_09-47-29](https://user-images.githubusercontent.com/129948909/230809605-efb8e5f2-5233-4636-8371-b1bb688f06c0.png)
, and it will continue to increase until the memory is insufficient.

**Reply by rodja** [reply]
Could you try to [allow more swap memory](https://superuser.com/questions/793304/how-to-increase-swap-memory-in-windows)?
